src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -61,24 +73,27 @@ export default class ResultsViewPage extends React.Component<IResultsViewPagePro
 
     public exposeComponentRenderersToParentScript(){
 
-        exposeComponentRenderer('renderMutationsTab', (props:{genes:string[], studyId:string, samples:string[]|string})=>{
-            const resultsViewPageStore = new ResultsViewPageStore();
-            resultsViewPageStore.hugoGeneSymbols = props.genes;
-            resultsViewPageStore.studyId = props.studyId;
-            if (typeof props.samples === "string") {
-                resultsViewPageStore.sampleListId = props.samples;
-            } else {
-                resultsViewPageStore.sampleList = props.samples;
-            }
+        exposeComponentRenderer('renderMutationsTab',
+            (props:MutationsTabInitProps)=>{
+                resultsViewPageStore.hugoGeneSymbols = props.genes;
+                if (props.studyToSampleMap) {
```

Comment:
seems like these modes are exclusive, so should setting one clear the other automatically?  Like, store should not be allowed to have  studyToSampleMap and studyToSampleIds?  We did this with patient view.  You can't have both patientId and sampleId set at once.